templates/index.html

```diff
@@ -3,6 +3,22 @@
 
 <head>
 
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-166920629-3"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'UA-166920629-3');
+  </script>
+
+  <script>
+    // temporary until Heroku bug gets sorted out
+    if (window.location.href.includes('heroku')) {
+      window.location.replace('http://ec2-3-237-25-198.compute-1.amazonaws.com:5000/');
```

Comment:
Please make this url an environment variable

Reply:
https://github.com/MLDSAI/VisionTracing/pull/2/commits/c9e4704ac7fc4c9e22947d03db8dbc368affa643